src/main/java/eu/dissco/core/translator/terms/media/MediaType.java

```diff
@@ -29,13 +29,13 @@ public String retrieveFromDWCA(JsonNode unit) {
 
   private String parseToOdsType(String recoveredType) {
     switch (recoveredType) {
-      case STILL_IMAGE, "Image" -> {
+      case "StillImage", "Image" -> {
```

Comment:
this would fit well as an enum (with a `fromString()` method), especially as we accept more data types. add US?

Reply:
Not completly sure what you mean, create an enum from the list that we have here?

Reply:
yeah exactly, it could be useful as we add more data types. Could move the string logic to the enum. But that might be more atomicity than necessary.